adapters/postgres/postgres.go

```diff
@@ -160,6 +149,27 @@ func JoinByRequest(r *http.Request) (values []string, err error) {
 	return joinValues, nil
 }
 
+func SelectByRequest(r *http.Request) []string {
```

Comment:
@marioidival I did this to avoid to parse when need to check the `_select` fields against the permitted fields, the old way will need handle the string, remove the "SELECT", "FROM" and separate the fields